src/components/UI/Layout/Navigation/SideDrawer/SideDrawer.tsx

```diff
@@ -243,34 +243,38 @@ export const SideDrawer: React.SFC<SideDrawerProps> = () => {
             >
               <Menu menus={staffManagementMenus}>
                 <Tooltip title="Staff Management" placement="top">
-                  <IconButton data-testid="staffManagementMenu">
-                    <img
-                      src={
-                        ['/group', '/staff-management', '/blocked-contacts'].includes(
-                          location.pathname
-                        )
-                          ? ActiveStaffIcon
-                          : InactiveStaffIcon
-                      }
-                      className={styles.StaffIcon}
-                      alt="staff icon"
-                    />
-                  </IconButton>
+                  <span>
```

Comment:
@rathorevaibhav SPAN is an inline element. This may work, but may not be the best practice.